daffodil-test/src/test/resources/org/apache/daffodil/section23/dfdl_functions/BitFunctions.tdml

```diff
@@ -0,0 +1,726 @@
+      <tdml:errors>
+        <tdml:error>Unsupported function: dfdlx:shiftLeft</tdml:error>
+      </tdml:errors>
+  </tdml:parserTestCase>
+
+  <tdml:parserTestCase name="rightShiftFloatError01" root="rightShiftFloat" model="BitFunctions">
+    <tdml:document>1.0,1</tdml:document>
+      <tdml:errors>
+        <tdml:error>Unsupported function: dfdlx:shiftRight</tdml:error>
+      </tdml:errors>
+  </tdml:parserTestCase>
+
+  <tdml:parserTestCase name="leftShiftDoubleError01" root="leftShiftDouble" model="BitFunctions">
+    <tdml:document>1.0,1</tdml:document>
+      <tdml:errors>
+        <tdml:error>Unsupported function: dfdlx:shiftLeft</tdml:error>
+      </tdml:errors>
+  </tdml:parserTestCase>
+
+  <tdml:parserTestCase name="rightShiftDoubleError01" root="rightShiftDouble" model="BitFunctions">
+    <tdml:document>1.0,1</tdml:document>
+      <tdml:errors>
+        <tdml:error>Unsupported function: dfdlx:shiftRight</tdml:error>
+      </tdml:errors>
+  </tdml:parserTestCase>
+
+  <tdml:parserTestCase name="leftShiftIntegerError01" root="leftShiftInteger" model="BitFunctions">
+    <tdml:document>1.0,1</tdml:document>
+      <tdml:errors>
+        <tdml:error>Unsupported function: dfdlx:shiftLeft</tdml:error>
+      </tdml:errors>
+  </tdml:parserTestCase>
+
+  <tdml:parserTestCase name="rightShiftIntegerError01" root="rightShiftInteger" model="BitFunctions">
+    <tdml:document>1.0,1</tdml:document>
+      <tdml:errors>
+        <tdml:error>Unsupported function: dfdlx:shiftRight</tdml:error>
+      </tdml:errors>
+  </tdml:parserTestCase>
+
+  <tdml:parserTestCase name="leftShiftDecimalError01" root="leftShiftDecimal" model="BitFunctions">
+    <tdml:document>1.0,1</tdml:document>
+      <tdml:errors>
+        <tdml:error>Unsupported function: dfdlx:shiftLeft</tdml:error>
+      </tdml:errors>
+  </tdml:parserTestCase>
+
+  <tdml:parserTestCase name="rightShiftDecimalError01" root="rightShiftDecimal" model="BitFunctions">
+    <tdml:document>1.0,1</tdml:document>
+      <tdml:errors>
+        <tdml:error>Unsupported function: dfdlx:shiftRight</tdml:error>
+      </tdml:errors>
+  </tdml:parserTestCase>
+
+  <tdml:parserTestCase name="leftShiftNonNegativeIntegerError01" root="leftShiftNonNegativeInteger" model="BitFunctions">
+    <tdml:document>1.0,1</tdml:document>
+      <tdml:errors>
+        <tdml:error>Unsupported function: dfdlx:shiftLeft</tdml:error>
+      </tdml:errors>
+  </tdml:parserTestCase>
+
+  <tdml:parserTestCase name="rightShiftNonNegativeIntegerError01" root="rightShiftNonNegativeInteger" model="BitFunctions">
+    <tdml:document>1.0,1</tdml:document>
+      <tdml:errors>
+        <tdml:error>Unsupported function: dfdlx:shiftRight</tdml:error>
+      </tdml:errors>
+  </tdml:parserTestCase>
+
+
+
+  <tdml:parserTestCase name="leftShiftIntError02" root="leftInt" model="BitFunctions">
+     <tdml:document>2,32</tdml:document>
+       <tdml:errors>
+         <tdml:error>32</tdml:error>
```

Comment:
As above, I think it would be worth checking for multiple strings to make sure we are always getting a reasonable error message. 

```xml
<tdml:errors>
  <tdml:error>Schema Definition Error</tdml:error>
  <tdml:error>dfdlx:leftShift</tdml:error>
  <tdml:error>width</tdml:error>
  <tdml:error>xs:int</tdml:error>
  <tdml:error>32</tdml:error>
  <tdml:error>35</tdml:error>
</tdml:error>
```
Not sure we have to go this crazy, but the idea is that we look for important diagnostic information that should be in the message without worring about the actual message itself.

Same goes for the below negative tests.